scripts/server_scripts/ha_shutdown_signal.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2022 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+"""
+This script will be sending a mock shutdown signal to HA
```

Comment:
This script sends a signal right ? or coping it to HA container and then running from there ?

Reply:
This particular script is used to send a mock cluster shutdown message signal directly on message bus, so that HA can consume it and process further as if CSM is sending to initiate cluster stop. This has to be executed inside the HA container to achieve the above said. 
I've added this to server scripts, so that it will be copied from local to HA container and then execute.